milestone-3/frontend/src/components/RecipePage/Ingredients/Ingredients.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function Ingredients({ingredients}) {
+  return (
+    <div>
+      <hr/>
+      <h2>Ingredients</h2>
+      <ul>
+        {
+          ingredients.map(function(ingredient){
```

Comment:
This would be a good place to use an arrow function, but it is totally not required.